src/CrouchBlocker.cs

```diff
@@ -13,21 +13,30 @@
 
 namespace Celeste.Mod.AnarchyCollab2022.Content {
     [Tracked]
-    [CleanupRemove]
     [CustomEntity("AnarchyCollab2022/CrouchBlocker")]
     public class CrouchBlocker : Trigger {
+        protected static int GlobalBlockerCount = 0;
+        private static ILHook updateHook;
+        private static Hook duckingSetterHook;
+        private static On.Celeste.Player.hook_DashEnd dashEndHook;
+
         private bool blockGlobal;
-        private ILHook updateHook;
-        private Hook duckingSetterHook;
-        private On.Celeste.Player.hook_DashEnd dashEndHook;
 
         public CrouchBlocker(EntityData data, Vector2 offset) : base(data, offset) {
             blockGlobal = data.Bool("blockGlobal");
         }
 
         public override void Added(Scene scene) {
+            if (blockGlobal) { GlobalBlockerCount++; }
             base.Added(scene);
+        }
+
+        public override void Removed(Scene scene) {
```

Comment:
Are you sure this is always called (if e.g. the player exits the level while in a room with a global trigger?)

Reply:
added a `SceneEnd` override to fix this